### ISSUE TITLE
I tried -i option to load initial model but failed

### ISSUE THREAD

**Issue by robbine** [JohnLangford/vowpal_wabbit#1050]
I set "-i sgdpre.vwmodel" in the command below, and vw output seems wrong
cat part-00000.shuf | ../vowpalwabbit/vw -b 20         --l1 0.000001         --l2 0.000001        --passes 100        -i sgdpre.vwmodel        -d /dev/stdin        -f sgd.vwmodel        --cache_file temp.cache       --loss_function=logistic

finished run
number of examples = 0
weighted example sum = 0
weighted label sum = 0
average loss = -nan
total feature number = 0
vw: multiple occurrences

**Comment by arielf** [comment]
"multiple occurrences" means "When using `-i model` You shouldn't repeat options which are stored in the model on the command line."

Also note that `cat datafile | vw ... -d /dev/stdin` can be simplified to `vw datafile`
